app/backend/src/clj/teet/log.clj

```diff
@@ -0,0 +1,84 @@
+(ns teet.log
```

Comment:
We use timbre logging on the frontend as well. Should we unify and have the wrapper there as well?

Reply:
Fine by me. At first we could just expose the blessed set of functions from timbre. Later we could maybe have a frontend logging endpoint where we'd send `log/error`s to backend for being further logged into CloudWatch.